cross-platform/react-app/src/frontend/Screens/ModelScreen.tsx

```diff
@@ -189,12 +186,12 @@ export function ModelScreen(props: ModelScreenProps) {
     {
       label: toolsLabel,
       isTab: true,
-      popup: React.createElement(getToolsBottomPanelFunction(), {
-        key: "tools",
-        iModel,
-        // Close the Views bottom panel when a view is selected from it.
-        onToolClick: () => { tabsAndPanelsAPI.closeSelectedPanel(); },
-      }),
+      popup: React.createElement(bottomPanel ?? ToolsBottomPanel,
+        {
+          iModel,
+          // Close the Views bottom panel when a view is selected from it.
+          onToolClick: () => { tabsAndPanelsAPI.closeSelectedPanel(); },
```

Comment:
This is missing from CameraSampleToolsBottomPanel, isn't it?

Reply:
I'm not sure I understand. The props are passed to the bottom panel (either `bottomPanel` or `ToolsBottomPanel`.

Reply:
Sorry, my bad.